services/blockchain-indexer/shared/dataService/pos/validators.js

```diff
@@ -164,6 +164,7 @@ const loadAllPosValidators = async () => {
 	}
 };
 
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Why do we need this?

Reply:
We do not need this anymore. had added to test some way of doing unit test which did not work but forgot to remove. will remove it.